tsl/src/continuous_aggs/create.c

```diff
@@ -856,9 +856,11 @@ cagg_create(const CreateTableAsStmt *create_stmt, ViewStmt *stmt, Query *panquer
 		 * Until then the choice was made in favor of the most generic schema
 		 * that can be optimized later.
 		 */
+		char *func_schema =
```

Comment:
I would recommend adding a check for `InvalidOid`  and `NULL`. This should not happen, but if the function does not exist, we should handle it properly.

```suggestion
		Oid bucket_func_namespace = get_func_namespace(bucket_info->bucket_func->funcid);
		Ensure(bucket_func_namespace != InvalidOid, "namespace for funcid %d not found", bucket_info->bucket_func->funcid);
		char *func_schema = get_namespace_name(bucket_func_namespace);
		Ensure (func_schema != NULL, "unable to get namespace name for Oid %d", bucket_func_namespace);
```